src/commentmodel.cpp

```diff
@@ -541,8 +541,11 @@ void CommentModel::moreComments(int index, const QVariant &children)
     QHash<QString, QString> parameters;
     parameters["sort"] = getSortString(m_sort);
     parameters["link_id"] = linkMap.value("fullname").toString();
-    QStringList limitedChildrenList(children.toStringList().mid(0,50));
-    m_moremoreComments = children.toStringList().mid(50);
+    QStringList const childrenList = children.toStringList();
+    QStringList const limitedChildrenList = childrenList.mid(0,50);
+    if (childrenList.size() > 50) {
+        m_moremoreComments = childrenList.mid(50);
```

Comment:
is this really the cause of a crash?? This should have been found ages ago if mid() really needs to start from a valid index..